pykokkos/interface/parallel_dispatch.py

```diff
@@ -61,9 +61,9 @@ def parallel_for(*args, **kwargs) -> None:
         handled_args.name,
         handled_args.policy,
         handled_args.workunit,
+        "for",
```

Comment:
minor: do we want enum for operations (rather than strings)

Reply:
Yeah this should be done, probably in a later PR